docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

```diff
@@ -6,7 +6,10 @@ ms.date: 01/25/2018
 
 # Sideload Office Add-ins for testing
 
-You can install an Office Add-in for testing in an Office client running on Windows by either 1) using a shared folder catalog to publish the manifest to a network file share, or 2) [running the "**npm run sideload**" command from the root of the add-in project folder (**NOTE**: this method only works for Excel, Word and PowerPoint add-ins)](sideload-office-addin-using-sideload-command.md).
+You can install an Office Add-in for testing in an Office client running on Windows by one of the following methods:
+
+- using a shared folder catalog to publish the manifest to a network file share (instructions below)
+- [running the "**npm run sideload**" command from the root of the add-in project folder.](sideload-office-addin-using-sideload-command.md) [!NOTE] this method only works for Excel, Word, and PowerPoint add-ins).

```

Comment:
I believe Markdown notes have to be on their own lines with '>' preceding the [!NOTE] and text. Here's the example from the docs site.
```markdown
> [!NOTE]
> Information the user should notice even if skimming.
```

Reply:
"This" at the start of the note should also be capitalized.